content/docs/1.3.0/advanced-resources/deploy/customizing-default-settings.md

```diff
@@ -3,22 +3,24 @@ title: Customizing Default Settings
 weight: 1
 ---
 
-You may customize Longhorn's default settings when deploying it. You may specify, for example, `Create Default Disk With Node Labeled` and `Default Data Path` before starting Longhorn.
-
-This default setting is only for a Longhorn system that hasn't been deployed. It has no impact on an existing Longhorn system. The settings for any existing Longhorn system should be modified using the Longhorn UI _or_ by kubectl. Reference to the [knowledge base doc](https://longhorn.io/kb/troubleshooting-default-settings-do-not-persist/) for the details.
+You may customize Longhorn's default settings while installing or upgrading. You may specify, for example, `Create Default Disk With Node Labeled` and `Default Data Path` before starting Longhorn.
 
 The default settings can be customized in the following ways:
 
-- [Deploy](#deploy)
+- [Installation](#installation)
   - [Using the Rancher UI](#using-the-rancher-ui)
   - [Using the Longhorn Deployment YAML File](#using-the-longhorn-deployment-yaml-file)
   - [Using Helm](#using-helm)
-- [Upgrade](#upgrade)
+- [Update Settings](#update-settings)
```

Comment:
Can't use helm?

Reply:
You mean only change the settings' value and do upgrade , right? If YES, it is feasible.

Reply:
Yes, just do upgrade but actually it's update.

Reply:
Updated. Thank you.

Reply:
Can users use Rancher UI to update settings as well? I believe yes.

Reply:
Interesting. Yes, the upgrade to the same version is feasible via Rancher UI.